src/envs/base_env.py

```diff
@@ -14,6 +14,7 @@ class BaseEnv(abc.ABC):
 
     def __init__(self) -> None:
         self._current_state = DefaultState  # set in reset
+        self._current_task = DefaultTask  # set in reset
```

Comment:
should we clarify that this is only used for rendering? are there other potential use cases down the line?

Reply:
I don't have any use cases in mind right now, but it seems plausible that there would be other ones down the line, and I think that's fine API-wise. I'm going to leave it alone